ui/pages/institutional/custody/custody.js

```diff
@@ -604,7 +607,9 @@ const CustodyPage = () => {
                 ),
               );
 
-              dispatch(setSelectedAddress(firstAccountKey.toLowerCase()));
+              dispatch(
+                setSelectedInternalAccount(firstAccountKey.toLowerCase()),
```

Comment:
This seems not correct, as `firstAccountKey` is an address no?

Reply:
Yes, this should actually be the account id.